community/louvain_common.go

```diff
@@ -35,11 +35,80 @@ func Q(g graph.Graph, communities [][]graph.Node, resolution float64) float64 {
 	}
 }
 
+// Multiplex is a multiplex graph.
```

Comment:
Yes, that looks good. The ref linked in the first comment of the PR is also worthwhile.